packages/fuels-programs/src/contract.rs

```diff
@@ -462,13 +465,51 @@ pub struct ContractCallHandler<T: Account, D> {
     pub account: T,
     pub datatype: PhantomData<D>,
     pub log_decoder: LogDecoder,
+    pub retry_config: RetryConfig,
+}
+
+// Implement Clone for ContractCallHandler once all types are provided.
+impl<T: Account, D> Clone for ContractCallHandler<T, D> {
```

Comment:
I needed this until @hal3e  joined one of his PRs. It stayed like this because it didn't cause me a problem, so I forgot about it. Now we can do as you said. `Clone`